### PR TITLE
stars

### PR DESCRIPTION
**Pull request by SergeyShapovalenko**
- [DEMO LINK](https://SergeyShapovalenko.github.io/layout_stars/)
- [TEST REPORT LINK](https://SergeyShapovalenko.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="box stars--0">
```
Comment (loralevitska) [inline src/index.html:11]: Fix it everywhere
```suggestion
    <div class="stars stars--0">
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (loralevitska) [inline src/style.css:1]:
```suggestion
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Oops, something wrong with deploy. 
image
If you have some problems with deploy fill free to ask in fe_chat

File: src/index.html [hunk 3]
```diff
@@ -8,6 +8,52 @@
...
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class=" stars stars--1">
```
Comment (tetiana-ivanchenko) [inline src/index.html:19]:
```suggestion
    <div class="stars stars--1">
```

File: src/style.css (hunk 2 above)
Comment (tetiana-ivanchenko) [reply src/style.css:1]: fit it https://github.com/mate-academy/layout_stars/pull/2357/files#r1066189879

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**